openml-h2o/src/test/java/com/feedzai/openml/h2o/algos/ParametersTest.java

```diff
@@ -62,4 +75,83 @@ public void testParametersExist() {
         });
     }
 
+    /**
+     * Tests that nothing is filtered if the {@link water.api.schemas3.ModelParametersSchemaV3} contains no static field
+     * named {@code fields}.
+     *
+     * @since 1.0.7
+     */
+    @Test
+    public void noFieldsField() {
```

Comment:
I like these tests, but aren't we missing the happy path? The field exists and contains `A` and `B`, so your field `C` will be filtered